app/celery/service_callback_tasks.py

```diff
@@ -168,3 +194,25 @@ def create_complaint_callback_data(complaint, notification, service_callback_api
         "service_callback_api_bearer_token": service_callback_api.bearer_token,
     }
     return signing.encode(data)
+
+
+def create_returned_letter_callback_data(service_id, reported_date, service_callback_api):
+    from app.letters.rest import fetch_returned_letter_data
+
+    returned_letter_data = fetch_returned_letter_data(service_id, reported_date)
+
+    data = {
+        "notification_id": str(returned_letter_data[0]["notification_id"]),
```

Comment:
I might have got this wrong, but from looking at the code it appears that this `create_returned_letter_callback_data` function is called once per returned letter, gets all the returned letters for the relevant service and yet only ever returns the first one. So if a service had 5 returned letters, its callback endpoint would be hit 5 times but every time they would receive details of the first letter.

I have tried to test this locally to see if this is the case, but am getting SQLAlchemy errors from this function which I haven't had a chance to dig into. This needs some thorough local testing to check everything is working as expected.